File: src/scripts/unpack-favicon.ts
const source = './data/favicon_io.tar.xz';
const dest = './src/app/website/static/';

const command = new Deno.Command('tar', {
    args: ['-xJf', source, '-C', dest],
});

const { success } = await command.output();

if (success) {
    try {
        Deno.removeSync(`${dest}about.txt`);
    } catch (e) {
        if (!(e instanceof Deno.errors.NotFound)) throw e;
    }
}

File: src/scripts/unpack_favicon.go
package main

import (
	"os"
	"os/exec"
	"path"
)

var source = path.Join("data", "favicon_io.tar.xz")
var dest = path.Join("src", "goapp", "website", "static")

func UnpackFavicon() {
	cmd := exec.Command("tar", "-xJf", source, "-C", dest)
	err := cmd.Run()
	Fatal(err)
	os.Remove(path.Join(dest, "about.txt"))
}
